chapters/object_types/volumes.txt

```diff
@@ -77,3 +77,28 @@ surface. Implementations may limit opacity to a value slightly smaller than one.
 A possible implementation of a rendering algorithm for this volume type is given
 in <<transferFunction1D_algorithm>>.
 ====
+
+[[object_types_volume_transfer_function2d]]
+==== TransferFunction2D
+
+Extension `KHR_VOLUME_TRANSFER_FUNCTION2D`
+
+A 2D transfer function uses two separate scalar fields to determine the color and opacity of the
+sampled `value`. This allows for a more nuanced and detailed visualization of volume data compared
+to 1D transfer functions. The 2D transfer function volume is created by passing the subtype string
+`transferFunction2D` to <<object_types_volume, `anariNewVolume`>>. It supports the following parameters:
+
+.<<api_concepts_parameters, Parameters>> understood by the transferFunction2D volume.
+[cols="<3,<5,>4,<7",options="header,unbreakable"]
+|===================================================================================================
+| Name             | Type                           | Default | Description
+| value            |`SPATIAL_FIELD`                 |         | <<object_types_spatial_field, Spatial field>> used for the scalar values of the volume
+| valueRange       |`FLOAT32_BOX1` / `FLOAT64_BOX1` |  [0, 1] | sampled values of `value` are clamped to this range
```

Comment:
Do we also need a `gradientRange` to clamp the gradient, which is then mapped to the `y` dimension of the transfer function?
Alternatively `valueRange` could be a `BOX2`, but this is a bit hacky (since `value` really is a scalar).